redirect/routes.php

```diff
@@ -110,12 +110,20 @@ class Route {
         ],
         'tmmi'         => [
             'description' => 'Technik der Mensch inen Interaktion [Not Found]',
-            'moodle_id'   => '-1'
+            'moodle_id'   => '6662'
         ],
         'mgl2'         => [
             'description' => 'Mathematische Grundlagen 2 [Lorencka]',
             'moodle_id'   => '4130'
         ],
+        'ffi_mathe2'   => [
+            'description' => 'Mathematische Grundlagen2; Analysis',
```

Comment:
Sollte so wie Lorencka den Namen enthalten
```suggestion
            'description' => 'Mathematische Grundlagen2 [Rößl]; Analysis',
```